### DISCUSSION TITLE
Renovate docs about autodiscovery and token is imprecise

### DISCUSSION THREAD

**Discussion by azro352** [renovatebot/renovate#25289] [Suggest an Idea]
Hi

The doc states the following after using[autodiscover](https://github.com/renovatebot/renovate/blob/a1e5f548d148f96904ca505c08a4c11825f98446/lib/workers/global/autodiscover.ts#L46) but hasn't find any

> The account associated with your token does not have access to any repos

But having not found any repositories is not necessarily related to the token, it can be caused by the filter `autodiscoverTopics` or `autodiscoverNamespaces`

Would you agree to a sentence like : 

> The account associated with your token didn't allow to find any repositories. This can be caused by the token's permissions, or the options autodiscoverTopics or autodiscoverNamespaces 

I will make the PR for it if you're ok

Thanks

**Comment by rarkins** [comment]
Ideally it would be even more clever and change the message based on whether topics or namespaces were configured. PR welcome!

**Reply by azro352** [reply]
Here is it https://github.com/renovatebot/renovate/pull/25290
